packages/ui/VirtualList/VirtualList.js

```diff
@@ -276,7 +297,18 @@ VirtualList.propTypes = /** @lends ui/VirtualList.VirtualList.prototype */ {
 	verticalScrollbar: PropTypes.oneOf(['auto', 'visible', 'hidden'])
 };
 
-VirtualList.defaultProps = {
+VirtualList.defaultPropValues = virtualListDefaultProps;
+
+/**
```

Comment:
Please don't move this jsdoc.

Reply:
I didn't move the jsdocs code in anywhere, however git diff shown as if it is moved. I fixed to only add definition before jsdoc.